src/transformers/models/bit/configuration_bit.py

```diff
@@ -96,13 +103,14 @@ def __init__(
         self.embedding_size = embedding_size
         self.hidden_sizes = hidden_sizes
         self.depths = depths
-        self.stem_type = stem_type
         self.layer_type = layer_type
         self.hidden_act = hidden_act
+        self.num_groups = num_groups
         self.drop_path_rate = drop_path_rate
         self.output_stride = output_stride
         self.width_factor = width_factor
-        self.conv_layer = conv_layer
+        self.embedding_dynamic_padding = embedding_dynamic_padding
+        self.global_padding = global_padding if global_padding is None else global_padding.upper()
```

Comment:
Maybe we can make this a Python enumerable?

Reply:
Proposed something to be consistent with `layer_type` in e03386b7ceabec4674a2455ff4df5b4381ab291f

Reply:
let me add another checker